phenol-core/src/test/java/org/monarchinitiative/phenol/ontology/testdata/hpo/HpoOntologyTestBase.java

```diff
@@ -0,0 +1,126 @@
+package org.monarchinitiative.phenol.ontology.testdata.hpo;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSortedMap;
+import com.google.common.collect.Lists;
+import org.monarchinitiative.phenol.ontology.data.*;
+
+import java.util.AbstractMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class HpoOntologyTestBase {
+  protected static final Ontology ontology;
+  protected static final TermId PHENOTYPIC_ABNORMALITY = TermId.of("HP:0000118"); //
+  // Heart
+  protected static final TermId CARDIOVASCULAR_ID = TermId.of("HP:0001626");
+  protected static final TermId ABN_CARDIOVASCULAR_PHYSIOLOGY = TermId.of("HP:0011025");
+  protected static final TermId ABN_HEART_SOUND =  TermId.of("HP:0031657");
+  protected static final TermId HEART_MURMUR =  TermId.of("HP:0030148");
+  protected static final TermId GALLOP_RHYTHM =  TermId.of("HP:0033113");
+  protected static final TermId SYNCOPE =  TermId.of("HP:0001279");
+  protected static final TermId VASOVAGAL_SYNCOPE =  TermId.of("HP:0012668");
+  protected static final TermId ORTHOSTATIC_SYNCOPE =  TermId.of("HP:0012670");
+  // EYE
+  protected static final TermId EYE_ID = TermId.of("HP:0000478");
+  protected static final TermId ABN_EYE_MORPHOLOGY = TermId.of("HP:0012372");
+  protected static final TermId COLOBOMA = TermId.of("HP:0000589");
+  protected static final TermId RETINAL_COLOBOMA = TermId.of("HP:0000480");
+  protected static final TermId IRIS_COLOBOMA = TermId.of("HP:0000612");
+  protected static final TermId ABN_GLOBE_LOCATION = TermId.of("HP:0100886");
+  protected static final TermId PROPTOSIS = TermId.of("HP:0000520");
+  protected static final TermId HYPERTELORISM = TermId.of("HP:0000316");
+
+  protected static final ImmutableMap.Builder<TermId, Term> termMapBuilder = ImmutableMap.builder();
+
+  protected static final ImmutableMap.Builder<Integer, Relationship> relationMapBuilder = ImmutableMap.builder();
+
+  protected static final List<ToyHpoAnnotation> hpoAnnotations;
+
+  protected static final AtomicInteger counter = new AtomicInteger(0);
```

Comment:
Does this need to be protected? I think it can be private.

Reply:
agree, done!